playbooks/aws/openshift-cluster/provision.yml

```diff
@@ -14,12 +14,12 @@
 
 - import_playbook: provision_elb.yml
 
-- name: Create the master node group
+- name: Create the master instances
   hosts: localhost
   tasks:
-  - name: provision cluster
+  - name: provision instances
     import_role:
       name: openshift_aws
-      tasks_from: provision.yml
```

Comment:
roles/openshift-aws/tasks/provision.yml seems to be unused after this change; please remove the file.